api/nakadi-event-bus-api.yaml

```diff
@@ -1245,6 +1245,121 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /event-types/{name}/timelines:
+    post:
+      tags:
+        - timelines-api
+      description: |
+        Creates a new timeline for an event type and makes it active.
+        The oauth resource owner username has to be equal to 'nakadi.oauth2.adminClientId' property
+        to be able to access this endpoint.
+      parameters:
+        - name: name
+          in: path
+          description: Name of the EventType.
+          type: string
+          required: true
+        - name: timeline_request
+          in: body
+          schema:
+            description: Storage id to be used for timeline creation.
+            type: object
+            properties:
+              storage_id:
+                type: string
+          required: true
+      responses:
+        '201':
+          description: New timeline is created and in use.
+        '403':
+          description: Access forbidden
+          schema:
+            $ref: '#/definitions/Problem'
+    get:
+      tags:
+        - timelines-api
+      description: |
+        List timelines for a given event type.
+        The oauth resource owner username has to be equal to 'nakadi.oauth2.adminClientId' property
+        to be able to access this endpoint.
+      parameters:
+        - name: name
+          in: path
+          description: Name of the EventType to list timelines for.
+          type: string
+          required: true
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: array
+            description: list of timelines.
+            items:
+              type: object
+              properties:
+                id:
+                  type: string
+                  format: uuid
+                event_type:
+                  type: string
+                order:
+                  type: integer
+                storage_id:
+                  type: string
+                topic:
+                  type: string
+                created_at:
+                  type: string
+                  format: date-time
+                switched_at:
+                  type: string
+                  format: date-time
+                cleanedup_at:
+                  type: string
+                  format: date-time
```

Comment:
It is not clear if it is more related to implementation, rather than to user view of it